docs/components/LessStyleGuideView.jsx

```diff
@@ -1,14 +1,233 @@
+            written with a BEM notation.</p>
+
+            <Example
+              title="Variables"
+              code={`
+                  @zIndex--1: 100;
+              `}
+            >
+              Variables are the most atomic building blocks of Dewey. Each variable defines a single value for a
+              specific attribute. Variables are lower-cased camelCase with a double dash <code>--</code> for modifiers.
+            </Example>
+
+            <Example
+              title="Mixins"
+              code={`
+                .text--bold {
+                   font-weight: bold;
+                   font-style: normal;
+                }
+              `}
+            >
+              Mixins are the next layer of Dewey, they assemble attributes into meaningfully grouped elements
+              that are readable and composable. Mixins are lower-cased camelCase with a double dash <code>--</code>
+              between modifiers.
+            </Example>
+
+            <Example
+              title="Mixin functions"
+              code={`
+                .border--x--s(@color) {
+                  .border--left--s(@color);
+                  .border--right--s(@color);
+                }
+              `}
+            >
+              We use mixin functions for more complex attribute collections that should be similarly reusable and
+              composable. Mixin functions are lower-cased camelCase with a double dash <code>--</code>
+              between modifiers.
+            </Example>
+
+            <Example
+              title="Components"
+              code={`
+                .TabBar--Tab--selected {
+                  .text--medium;
+                  .border--bottom--m(@primaryBlue);
+                  box-sizing: border-box;
+                  color: @primaryBlue;
+                  cursor: pointer;
+                }
+              `}
+            >
+              Components are composed of variables and mixins to define interactions for specific elements and UI
+              patterns. Components use a title-cased CamelCase with a double dash <code>--</code> between
+              modifiers and descendants.
+            </Example>
+
+            <h3>IDs vs. Classes</h3>
+            <p>You should almost never need to use IDs. Broken behavior due to ID collisions are hard to track
+            down and annoying.</p>
+
+            <h2>Attributes and Units</h2>
+            <p>Dewey is designed to be a scalable and maintainable LESS system. We do this by using relying
+            heavily on mixins and variables to declare attributes and their values so that if something
+            changes at the base level with a variable or mixin, that change cascades correctly throughout
+            the system.</p>
+
+            <p>For sizing we have sizing variables that are set in REM units. Where a sizing variable doesn't
+            exist to fit your needs use REM units, or in rare cases EM units.</p>
```

Comment:
Most engineers (including me) don't understand the difference between REM and EM units. Maybe a 1-2 sentence explanation of how they are different, and why they are preferable to `px` measurements?